html/semantics/embedded-content/the-iframe-element/cross-origin-to-whom.window.js

```diff
@@ -0,0 +1,40 @@
+async_test(t => {
+  const frame = document.body.appendChild(document.createElement("iframe"));
+  frame.src = "support/document-with-embedded-svg.html";
+  const elements = {
+    "embed": ["getSVGDocument"],
+    "frame": ["contentDocument"],
+    "iframe": ["getSVGDocument", "contentDocument"],
+    "object": ["getSVGDocument", "contentDocument"]
+  };
+  function assert_apis(instance) {
+    const name = instance.localName;
+    let priorPossibleDocument = null;
+    elements[name].forEach(api => {
+      let possibleDocument = null;
+      if (api == "getSVGDocument") {
+        possibleDocument = instance[api]();
+      } else {
+        possibleDocument = instance[api];
+      }
+      assert_not_equals(possibleDocument, null, `${name}[${api}]`);
```

Comment:
Nit:

```suggestion
      assert_not_equals(possibleDocument, null, `${name}.${api}`);
```

(here and in the other file)